src/enrich/lib/services/redis.js

```diff
@@ -12,13 +12,18 @@ const redisClient = redis.createClient({
 redisClient.get = util.promisify(redisClient.get);
 redisClient.ping = util.promisify(redisClient.ping);
 
+/**
+ * Ping redis service.
+ *
+ * @returns {Boolean} Ping
```

Comment:
Do not confuse `Boolean` with `boolean` (same as `String` and `string`, cf. https://www.geeksforgeeks.org/what-is-the-difference-between-string-and-string-in-typescript/)
```suggestion
 * @returns {boolean} Is ping sucessful
```